webui/src/pages/auth/reset-password.jsx

```diff
@@ -0,0 +1,188 @@
+import React, {useRef, useState} from "react";
+import Layout from "../../lib/components/layout";
+import Row from "react-bootstrap/Row";
+import Card from "react-bootstrap/Card";
+import Form from "react-bootstrap/Form";
+import Col from "react-bootstrap/Col";
+import Button from "react-bootstrap/Button";
+import {
+    AlertIcon,
+    MailIcon
+} from "@primer/octicons-react";
+import {Error} from "../../lib/components/controls";
+import {auth} from "../../lib/api";
+import validator from "validator/es";
+
+const TOKEN = "token";
+
+const RequestResetPasswordForm = () => {
+
+    const onEmailChange = () => {
+        if (validator.isEmail(email.current.value)) {
+            setFormValid(true);
+            setEmailValid(true);
+        } else {
+            setFormValid(false);
+            setEmailValid(false);
+        }
+    };
+
+    const [reqResetPwdError, setReqResetPwdError] = React.useState(null);
+    const [riskAccepted, setRiskAccepted] = React.useState(false);
+    const [resetReqSent, setResetReqSent] = React.useState(false);
+    const [formValid, setFormValid] = useState(false);
+    const [emailValid, setEmailValid] = useState(null);
+    const email = useRef(null);
+
+    if (resetReqSent) {
+        return (
+            <Row>
+                <Col md={{offset: 4, span: 4}}>
+                    <ResetEmailSent/>
+                </Col>
+            </Row>
+        );
+    } else {
+        return (
+            <Row>
+                <Col md={{offset: 4, span: 4}}>
+                    <div className="reset-pwd-risks">
+                        <div className="risk-icon">
+                            <AlertIcon size={20}/>
+                        </div>
+                        <div className="warn-message">
+                            We don&#39;t store plain text passwords, therefore, once reset, they&#39;ll be lost.<br/>
```

Comment:
Nit, opinion: no need for all this info: 
1. It goes without saying that we don't store them.
2. If they don't have access to the mailbox, they can't reset the password.
3. Reset link is available for six hours - this info should be in the email itself

Reply:
It's not the first time I'm hearing this, so I think that we can drop this text and if we'll have any complaints or misalignments, we'll figure out what text to our here.
At first, this warning was for the application credentials, this is why the warning was added in the first place.

Reply:
Cool, I removed it